tests/acceptance/features/apiSharingNg/sharedWithMe.feature

```diff
@@ -2974,4 +2974,514 @@ Feature: an user gets the resources shared to them
         }
       }
     }
-    """
\ No newline at end of file
+    """
+
+
+  Scenario: user lists the space file shared with them when auto-sync is disabled
+    Given the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "projectSpace" with the default quota using the Graph API
+    And user "Alice" has uploaded a file inside space "projectSpace" with content "to share" to "textfile.txt"
+    And user "Brian" has disabled the auto-sync share
+    And user "Alice" has sent the following share invitation:
+      | resource        | textfile.txt |
+      | space           | Personal     |
```

Comment:
```suggestion
      | space           | projectSpace     |
```

Reply:
tests are passing even when the space is wrong? :thinking: 
Maybe json scema assertion is not working as expected?

Reply:
Oh! I forget that step